### PR TITLE
ACA-1662 - Enhance integration tests for deploy_ovf role

### PR DESCRIPTION
**Pull request by elsapassaro**
Jira ticket [ACA-1662](https://issues.redhat.com/browse/ACA-1662)

### CODE REVIEW

File: tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml
Comment (mikemorency) [inline tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml:1]: this portion of the test seems unrelated to the cloud.vmware_ops repo, i am not sure we should include it here?

Comment (elsapassaro) [reply tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml:1]: I'm using it in the setup step in order to create an OVF file in a content library, so we can test also the case of creating a VM using this OVF file from the content library

Comment (mikemorency) [reply tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml:1]: oh, i see. I misread the final task in this file and thought it was deploying the OVF not creating it

**Review by mikemorency — APPROVED** [review]

File: tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- name: Create Content Library
+  community.vmware.vmware_content_library_manager:
+    hostname: "{{ deploy_ovf_hostname }}"
+    username: "{{ deploy_ovf_username }}"
+    password: "{{ deploy_ovf_password }}"
+    validate_certs: false
+    library_name: "{{ ovf_library }}"
+    library_type: local
+    datastore_name: datastore3
```
[the remark this answers is not in the document]
Comment (elsapassaro) [reply tests/integration/targets/vmware_ops_deploy_ovf_test/tasks/export_vm_as_ovf_content_library.yml:20]: right, thanks!
